capstone-backend/config.js

```diff
@@ -0,0 +1,9 @@
+const PARSE_APP_ID = process.env.PARSE_APP_ID;
+const PARSE_JAVASCRIPT_KEY = process.env.PARSE_JAVASCRIPT_KEY;
+const MASTER_KEY = process.env.MASTER_KEY;
```

Comment:
These three const are not used in the file. The exports hold the hardcoded config values already. 
I bet the app would still work exactly the same if we remove these three lines.